bazel/setup_clang.sh

```diff
@@ -28,7 +28,7 @@ build:clang --linkopt='-L$(llvm-config --libdir)'
 build:clang --linkopt='-Wl,-rpath,$(llvm-config --libdir)'
 
 build:clang-asan --action_env=ENVOY_UBSAN_VPTR=1
-build:clang-asan --copt=-fsanitize=vptr,function
+# build:clang-asan --copt=-fsanitize=vptr,function
```

Comment:
not sure why but this breaks the opentracing cmake build

Reply:
```console
    CMakeFiles/cmTC_c1a95.dir/testCXXCompiler.cxx.o:(.data.rel.ro._ZTIFivE[_ZTIFivE]+0x0): undefined reference to `vtable for __cxxabiv1::__function_type_info'
```

https://dev.azure.com/cncf/envoy/_build/results?buildId=136477&view=logs&j=1439b9f7-a348-5b50-b5fe-ea612ea91241&t=20bd2bcd-3fb5-5bf3-9cdb-ca6f94dd3096&l=133